inst/assets/js/shinyfeedback.js

```diff
@@ -360,12 +360,15 @@
   // return the element containing the shiny inputId
   function findInput(inputId) {
     // from https://github.com/daattali/advanced-shiny/blob/master/update-input/www/app-shinyjs.js
-    // Escape characterss that have special selector meaning in jQuery
+    // Escape characters that have special selector meaning in jQuery
     inputId = inputId.replace( /(:|\.|\[|\]|,)/g, "\\$1" );
```

Comment:
I think we should make an `escape_id()` JS function since this non trivial regex is used in multiple places.  Can you add that?

Reply:
Resolved w/ 05e1378